concepticondata/concepticon.tsv

```diff
@@ -3060,3 +3060,58 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
+3082	SNOWMAN	Basic actions and technology	A mass of snow sculptured into the shape of a human.	Person/Thing	
+3083	SOFA	The house	Piece of furniture usually stuffed and used for seating two or more persons.	Person/Thing	
+3084	SUNGLASSES	Clothing and grooming	Form of glasses with tinted lenses to protect one's eyes from sunlight.	Person/Thing	
+3085	SWING	Basic actions and technology	Tool used for play with a seat hanging from ropes.	Person/Thing	
+3086	TANK	Warfare and hunting	Armoured vehicle used by the military.	Person/Thing	
+3087	TENNIS BALL	Modern world	Yellow, spherical object used for playing the game of tennis.	Person/Thing	
+3088	THERMOMETER	Basic actions and technology	Instrument used for measuring temperature, mostly by means of a contracting or expanding substanced contained within the instrument. 	Person/Thing	
+3089	UMBRELLA	Clothing and grooming	Portable folding canopy used for protection against weather influences.	Person/Thing	
+3090	WARDROBE	The house	Large object, often made out of wood, used for storing clothes or other fabrics.	Person/Thing	
+3091	WHISTLE (INSTRUMENT)	Basic actions and technology	Wind instrument most often made out of metal or wood used for producing a loud, shrill tone.	Person/Thing	
+3092	BOX (SOMEBODY OR SOMETHING)	Basic actions and technology	The act of using fists to fight or hit.	Action/Process	
+3093	BRUSH (TEETH)	Clothing and grooming	Clean one's teeth by repeatedly grating over them with an object.	Action/Process	
+3094	CLAP	Motion	Make noise by putting one's hand together.	Action/Process	
+3095	CONDUCT	Speech and language	To lead some kind of musical performance.	Action/Process	
+3096	ERUPT	The physical world	Burst of matter from a volcano.	Action/Process	
+3097	FILE	Clothing and grooming	Make something smooth, often nails or other parts of the body.	Action/Process	
+3098	GRATE	Basic actions and technology	To abrade an object by rubbing it against a rough surface.	Action/Process	
+3099	HAMMER (SOMETHING)	Basic actions and technology	Strike or beat something with a hammer.	Action/Process	
+3100	HATCH	Animals	Emergence from an egg.	Action/Process	
+3101	HITCHHIKE	Motion	Travel by means of lifts in vehicles.	Action/Process	
+3102	HUG	Social and political relations	The act of embracing someone or something with one's arms.	Action/Process	
+3103	IRON (CLOTHES)	Clothing and grooming	To make clothes smooth by brushing over them with a heated tool.	Action/Process	
+3104	MASSAGE	Basic actions and technology	To rub someone's body with the hands.	Action/Process	
+3105	MOP	Basic actions and technology	To clean or wipe up.	Action/Process	
+3106	OPERATE (MEDICINE)	The body	Perform a medical procedure by cutting or making incisions.	Action/Process	
+3107	PAINT (SOMETHING)	Basic actions and technology	The act of spreading coloured pigments on a surface.	Action/Process	
+3108	URINATE	The body	Excrete urine from the body.	Action/Process	
```

Comment:
we have PEE, or PISS already

Reply:
btw: the reason why you don't find it is: I have not recreated the database, since @tresoldi had similar problems, so with his PR, we actually HAVE the linking between "urinate" and "piss" now, which is similar to the "eggplant" case.

Reply:
Duh, sorry.